src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -83,41 +83,58 @@ public AsymmetricSignatureProvider(AsymmetricSecurityKey key, string algorithm,
                 throw new ArgumentOutOfRangeException("key.KeySize", key.KeySize, string.Format(CultureInfo.InvariantCulture, LogMessages.IDX10630, key.GetType(), SignatureProviderFactory.MinimumAsymmetricKeySizeInBitsForVerifying));
             }
 
+#if DNXCORE50
+            hash = GetHashAlgorithmName(algorithm);
+#else
             hash = GetHashAlgorithm(algorithm);
+#endif
             RsaSecurityKey rsaKey = key as RsaSecurityKey;
+
             if (rsaKey != null)
             {
+#if DNXCORE50
+                rsaCng = new RSACng();
```

Comment:
Nope :frowning:.  https://github.com/dotnet/corefx/issues/2953, if you want to watch for progress.